include/clover_field.h

```diff
@@ -109,12 +109,12 @@ namespace quda {
     bool reconstruct
       = clover::reconstruct(); /** Whether to create a compressed clover field that requires reconstruction */
     bool inverse = true;       /** Whether to create the inverse clover field */
+    CloverField *field = nullptr; /** Pointer to another field instance from which we can clone */
     void *clover = nullptr;    /** Pointer to the clover field */
     void *cloverInv = nullptr; /** Pointer to the clover inverse field */
     double csw = 0.0;          /** C_sw clover coefficient */
     double coeff = 0.0;        /** Overall clover coefficient */
     QudaTwistFlavorType twist_flavor = QUDA_TWIST_INVALID; /** Twisted-mass flavor type */
-    bool twisted = false;                                  /** Whether to create twisted mass clover */
```

Comment:
Just confirming my understanding: this can be deleted because it's redundant of what `twist_flavor` can encode?

Reply:
It can be deleted because it wasn't actually being used for anything 😄.  Yes, it was redundant.